src/components/GlobalNavigasjon.js

```diff
@@ -139,7 +105,7 @@ class GlobalNavigasjon extends Component {
                     const className = cn('navigasjonspanel', {
                         'navigasjonspanel--aktiv': menypunkt === aktivtMenypunkt,
                     });
-                    const antallPrikker = finnAntallPrikker(menypunkt, oppgaver, motebehovReducer, moterReducer);
+                    const antallPrikker = finnAntallPrikker(menypunkt, motebehovReducer, moterReducer, oppfolgingsplanerReducer);
```

Comment:
Blir det vist riktig antall prikker på hver fane nå?
Sånn umiddelbart ser den ut som om det summes planer, motebehov og moteplaner og det vises for begge faner.

Reply:
Det er ikke sikkert at dette er en god måte å gjøre det på, men jeg sjekker hvilket menypunkt vi er på når prikkene skal vises frem.

Reply:
Ja, men antallPrikker= 2  om det er 1 prikks for plan og 1 prikk for motebehov.
Ser ut som om det da vises 2 prikker i menypunktet oppfolgingsplaner. Eller stemmer det ikke?

Reply:
Nei, finnAntallPrikker tar også høyde for menypunkt, så den teller ikke prikk for moter/motebehov hvis menypunktet er OP, og den teller ikke OP-prikker hvis menypunktet er møteplanlegger.